pandunia-lekse.csv

```diff
@@ -2252,31 +2264,31 @@ paladium|mate|Pd||palladium||paladio|||||||||||||paladio|palladium|pallad
 panamia|dex|PA||Panama||Panamá|||||||||||||Panamo|Panama|Panama
 panasi||||pan-Asian||panasiano||||||||||||||panaasialainen|panazjatycki
-pande||||panda||panda|||||||||||||pando|panda|panda
+pande|||eng:spa:por:fra:may:panda, rus:панда (panda), hin:पांडा (pāṇḍā), jpn:パンダ (panda)|panda||panda|||||||||||||pando|panda|panda
 pandunia||||Pandunia||Pandunia|||||パンドゥニア||||||||pandunia|pandunia|Pandunia
 panduniste||||Pandunia speaker||hablante de Pandunia||||||||||||||pandunian puhuja|osoba mówiąca w Panduni
 paneuropi||||pan-European||paneuropeo||||||||||||||paneurooppalainen|paneuropejski
 pang (pange)|||tha:ปัง (pang), kor:빵 (ppang), khm:នំប៉័ង (numpang), jpn:パン (pan), por:pão, spa:pan, fra:pain|bread|pain|pan|pão|хлеб|خُبْز|面包|パン|||रोटी (नान)||roti|mkate||pano|leipä|chleb
-panike||||panic||pánico|||||||||||||paniko|hätä (paniikki)|panika
+panike|||eng:panic, spa:pánico, por:pânico, fra:panique, rus:паника (panika), fas:پانیک‎ (pânik), jpn:パニック (panikku)|panic||pánico|||||||||||||paniko|hätä (paniikki)|panika
 panire|||hin:पनीर (panīr), pnb:ਪਨੀਰ (panīr), ben:পনির (pônir), tur:peynir, eng:paneer|quark (paneer)|fromage blanc|queso fresco|queijo fresco|творог|||クワルク|||||||||rahka|twaróg
 panjabi|nas|||Punjabi||punjabí||||||||||||||pandžabi (eräs intialainen kieli)|pendżabski
 panke|||hin:पंखा (paṅkhā), urd:(pankhā), mya:ပန်ကာ (panka), fas:(panke), + eng:punkah, fra:panka, deu:Pankha|fan (ventilator)||ventilador|||||||||||||ventumilo|viuhka (tuuletin)|wiatrak (wentylator)
-panodemi||||pandemic||pandémico||||||||||||||pandeeminen (koko väestöä koskeva)|pandemiczny
-panodemonia||||pandemonium||pandemonio||||||||||||||pandemonio|pandemonium
-panodewia||||pantheon||panteón||||||||||||||pantheon|panteon
-panodewistia||||pantheism||panteísmo||||||||||||||monijumalisuus (panteismi)|panteizm
-panoduni||||global (concerning the whole world)||global (de todo el mundo)|||||||||||||tuttera|maailmanlaajuinen (globaali)|globalny (światowy)
-panoromansi||||panromantic||panrromántico|||||||||||||||panromantyczny
-panoseksi||||pansexual||pansexual|||||||||||||||panseksualny
-pantalone||||pants (trousers)||pantalones|||||||||||||pantalonoj|housut|spodnie, pantalony, portki
+pandemi||||pandemic||pandémico||||||||||||||pandeeminen (koko väestöä koskeva)|pandemiczny
+pandemonia||||pandemonium||pandemonio||||||||||||||pandemonio|pandemonium
+pandewia||||pantheon||panteón||||||||||||||pantheon|panteon
+pandewistia||||pantheism||panteísmo||||||||||||||monijumalisuus (panteismi)|panteizm
+panduni||||global (concerning the whole world)||global (de todo el mundo)|||||||||||||tuttera|maailmanlaajuinen (globaali)|globalny (światowy)
+panromansi||||panromantic||panrromántico|||||||||||||||panromantyczny
+panseksi||||pansexual||pansexual|||||||||||||||panseksualny
+pantalone|||spa:pantalón, fra:pantalon, ara:بنطلون‎ (banṭalōn), hin:पैंट  (paiṇṭ)|pants (trousers)||pantalones|||||||||||||pantalonoj|housut|spodnie, pantalony, portki
```

Comment:
I propose shorter root, "pante". The etymology would be the same but "eng:pants, hin:पैंट (paiṇṭ)" would be first.